tests/test_cli.py

```diff
@@ -0,0 +1,53 @@
+"""Integration test for the swmmanywhere command-line interface."""
+
+from __future__ import annotations
+
+import sys
+import tempfile
+from pathlib import Path
+
+import yaml
+
+from swmmanywhere import __main__
+from swmmanywhere.logging import logger
+
+
+def test_swmmanywhere_cli():
+    """Test that the CLI can successfully run with an actual configuration."""
+    with tempfile.TemporaryDirectory() as tempdir:
+        base_dir = Path(tempdir)
```

Comment:
More elegant to use pytest's `tmp_path` fixture: https://docs.pytest.org/en/6.2.x/tmpdir.html

```suggestion
def test_swmmanywhere_cli(tmp_path):
    """Test that the CLI can successfully run with an actual configuration."""
```

and use `tmp_path` as your `base_dir`.

Reply:
oo didn't know about that! thanks!